torch/csrc/api/include/torch/nn/modules/embedding.h

```diff
@@ -13,19 +13,31 @@ namespace nn {
 /// Options for the `Embedding` module.
 struct TORCH_API EmbeddingOptions {
   EmbeddingOptions(int64_t count, int64_t dimension);
-  /// The number of embeddings (number of rows in the table).
+  // The number of embeddings (number of rows in the table).
```

Comment:
We should actually use `///` instead of `//` because we need all the parameter comments to show up as docs in https://pytorch.org/cppdocs/api/structtorch_1_1nn_1_1_embedding_options.html#exhale-struct-structtorch-1-1nn-1-1-embedding-options.

Reply:
Also I think we should change the comment to match the Python side: https://github.com/pytorch/pytorch/blob/9181b9c73e0772bc6d245f5f7f4a47771bef02bb/torch/nn/modules/sparse.py#L17